### DISCUSSION TITLE
Can we allow Search Result web part to show updated results from sharepoint without the need to refresh the page

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3927] [Q&A]
I have added a Search Result web part which is connected to Search Box, Filter and Vertical web parts, as follow:-

![image](https://github.com/user-attachments/assets/3396e056-b063-4f6d-aa84-091f2abfba6f)

now is there a way to force the Search Result web part to check for updated data from SharePoint behind the scenes? if this is not possible then can we add a Refresh icon to manually refresh the data without the need to do a full page refresh ?

Thanks

**Comment by kasperbolarsen** [comment]
No, that is not a feature in the current version. Feel free to submit a PR

**Reply by mvcsharepointdev** [reply]
@kasperbolarsen ok thanks for the reply, I were able to do this workaround, where I added a Vertical item, with the Refresh icon >> and it will link to the current page:-

![image](https://github.com/user-attachments/assets/4d63271b-d2f9-4d47-8a41-91efb7ff997c)


so atleast the user can click on the icon to get updated data instead of clicking on the browser refresh iron..
